pytorch_module_path: "/home/demo/Workspace/assembly_hole_segmentation"
weight_path: "/home/demo/catkin_ws/src/assembly_part_recognition/pretrained_weights/hole/best_Unet.pt"
width: 640
height: 400
num_classes: 5
min_depth: 1.0
max_depth: 2.3
grid_size: 3
gpu_id: "0"

